### PR TITLE
Improved definitions for PGGD-04 related forms

### PR DESCRIPTION
**Pull request by KellyVance**
Definition changes as suggested by @BlakePaul and @LizDerrington 
No content or structural changes.

vocab owners and smes @BlakePaul and @LizDerrington 
technical check @LukeHauck

### CODE REVIEW

**Review by BlakePaul — COMMENTED** [review]
I suggest that Abandoned have a notice about PGGD-04 mentioned to make it the same as all the other vocabs and a few other small changes for consistency. "A notice that the intended hydraulic fracturing activity was cancelled and abandoned, the activity was never undertaken and no fluid injection occurred. Lodgement of Form PGGD-04. Borehole status does not change."

**Review by LizDerrington — APPROVED** [review]

**Comment by LizDerrington** [comment]
I agree with Paul's suggested change in definition above.

File: vocabularies/borehole-status-event.ttl [hunk 1]
```diff
@@ -76,7 +76,7 @@ bhse:hydraulic-fracturing-on-notice a skos:Concept ;
     skos:topConceptOf <http://linked.data.gov.au/def/borehole-status-event> .
 
 bhse:hydraulic-fracturing-abandoned a skos:Concept ;
-    skos:definition "Recording that the intended hydraulic fracturing activity was cancelled and abandoned and the activity was never undertaken. Borehole status will not change."@en ;
+    skos:definition "Records that the intended hydraulic fracturing activity was cancelled and abandoned, the activity was never undertaken and no fluid injection occurred. Borehole status does not change."@en ;
```
Comment (KellyVance) [inline vocabularies/borehole-status-event.ttl:79]:
```suggestion
    skos:definition "A Notice that the intended hydraulic fracturing activity was cancelled and abandoned, the activity was never undertaken and no fluid injection occurred. Lodgement of Form PGGD-04. Borehole status does not change"@en ;
    
    @BlakePaul change to this?

```

**Review by LukeHauck — APPROVED** [review]
Tech good. Approved.

**Review by DavidCrosswellGSQ — APPROVED** [review]

**Pull request merged**